docs/en/CSMM/self-host/installation.md

```diff
@@ -103,7 +103,7 @@ DBSTRING, a [special syntax](https://sailsjs.com/documentation/reference/configu
 
 CSMM_HOSTNAME is used to generate links, CORS and other things. You should set this to the ip/domain you will access CSMM from. Do NOT add a trailing slash to this.
 
-Redis configuration uses a similar system to the MySQL connection string. redis://[:password]127.0.0.1:6379
+Redis configuration uses a similar system to the MySQL connection string. redis://:password@127.0.0.1:6379
```

Comment:
Maybe we can add a line saying the password is not always required (depending on if you've set a password on Redis or not)

Might be worth it to provide both examples.

```
redis://:password@127.0.0.1:6379
redis://127.0.0.1:6379
```
Not sure if that will make it clearer, WDYT?

Reply:
Yeah I like it, made a change that reflects this 👍